### PR TITLE
visual fixes

### PR DESCRIPTION
**Pull request by Fire-Chimp**
visual fixes

### CODE REVIEW

**Review by ziang4891 — CHANGES_REQUESTED** [review]
Duplicated file with ''43495 Gregoria Gurog Destroyer.sql' :: no comma

File: Database/Patches/2010-11-FromDarknessLight/9 WeenieDefaults/Creature/Human/43804 Sir Donovan.sql [hunk 1]
```diff
@@ -180,7 +166,7 @@ VALUES (@parent_id, 0, 12 /* TurnToTarget */, 0, 1, NULL, NULL, NULL, NULL, NULL
...
      , (@parent_id, 4, 18 /* DirectBroadcast */, 1.5, 1, NULL, 'Sir Donovan pulls out some salvaged supplies, and converts a section of the shard into a rather delicate looking wand.', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)
-     , (@parent_id, 5, 49 /* AwardLevelProportionalXP */, 1, 1, NULL, NULL, NULL, 0, 280000000, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 0.19999999, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)
+     , (@parent_id, 5, 49 /* AwardLevelProportionalXP */, 1, 1, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 0.19999999, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)
```
Comment (ziang4891) [inline Database/Patches/2010-11-FromDarknessLight/9 WeenieDefaults/Creature/Human/43804 Sir Donovan.sql:169]: Removes the XP gain settings

**Review by ziang4891 — CHANGES_REQUESTED** [review]
43847 Sir Learth.sql and 43988 Lady Daenerah.sql are already in 2011-03-HiddenInShadows, which will overwrite anything put in these two files in this PR

**Comment by Fire-Chimp** [comment]
On these two that are in November, do I put them in November then?

**Comment by ziang4891** [comment]
Yes, any changes should be applied against the ones in November patch folder

**Comment by Fire-Chimp** [comment]
I had those in November

**Comment by Fire-Chimp** [comment]
Oops never mind

File: Database/Patches/2010-11-FromDarknessLight/9 WeenieDefaults/Creature/Human/43804 Sir Donovan.sql (hunk 1 above)
Comment (ziang4891) [reply Database/Patches/2010-11-FromDarknessLight/9 WeenieDefaults/Creature/Human/43804 Sir Donovan.sql:169]: Change is still Null'ing the AwardLevelProportionalXP values

File: Database/Patches/2011-03-HiddenInShadows/9 WeenieDefaults/Creature/Human/43988 Lady Daenerah.sql [hunk 2]
```diff
@@ -0,0 +1,191 @@
+DELETE FROM `weenie` WHERE `class_Id` = 43988; 
+
```
Comment (ziang4891) [inline Database/Patches/2011-03-HiddenInShadows/9 WeenieDefaults/Creature/Human/43988 Lady Daenerah.sql:2]: Should be moved to the 2011-04-DarkDeliveries Patches subfolder

**Review by ziang4891 — CHANGES_REQUESTED** [review]

**Comment by Fire-Chimp** [comment]
I hope I did it right

**Comment by Fire-Chimp** [comment]
I am kinda confused on sir Donovan. All I did was changes his underclothes. I did nothing to the awardlevelproportionaxp values.

File: Database/Patches/2010-11-FromDarknessLight/9 WeenieDefaults/Creature/Human/43804 Sir Donovan.sql (hunk 1 above)
Comment (Fire-Chimp) [reply Database/Patches/2010-11-FromDarknessLight/9 WeenieDefaults/Creature/Human/43804 Sir Donovan.sql:169]: let me know if that didnt fix it.

**Review by ziang4891 — APPROVED** [review]

**Pull request merged**